src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketRepositorySource.java

```diff
@@ -34,13 +35,15 @@
 public class BitbucketRepositorySource {
     private final String path;
     private final String type;
+    private final String hash;
     private final List<String> attributes;
 
     @JsonCreator
-    public BitbucketRepositorySource(@JsonProperty("path") String path, @JsonProperty("type") String type, @JsonProperty("attributes") List<String> attributes) {
+    public BitbucketRepositorySource(@JsonProperty("path") String path, @JsonProperty("type") String type, @JsonProperty("attributes") List<String> attributes, @JsonProperty("commit") Map commit) {
```

Comment:
signature change

Reply:
It is a change in signature but running query `https://github.com/search?utf8=✓&q=org%3Ajenkinsci+BitbucketRepositorySource&type=Code` doesn't result in any uses outside of bitbucket-branch-source plugin. Also intent of this class is purely to deserialize bitbucket repository api json response in to java class so change in signature is not that big a risk. Perhaps it should be annotated with `@Restricted(NoExternalUse.class)`.